README.md

```diff
@@ -0,0 +1,103 @@
+# godockerize
+
+godockerize builds a Dockerfile for a given Go project. The base image is Alpine
+3.8.
```

Comment:
It might be good to omit the `3.8` here. I can see someone forgetting to update the readme when the base version is bumped.